check-storage-usage/CheckStorageUsage.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Search.Documents.Indexes;
+using Azure.Search.Documents.Indexes.Models;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace check_storage_usage
+{
+    public class CheckStorageUsage
+    {
+        // Run on a timer
+        // https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-timer?tabs=in-process&pivots=programming-language-csharp#ncrontab-expressions
```

Comment:
For line 17, can you get rid of the en-us in the URL?

For line 19, is this timer every 30 minutes?  If yes, could mention it.

Reply:
sure